server/api.js

```diff
@@ -1,11 +1,18 @@
 'use strict'
-
+const {Thing} = require('APP/db')
 const api = module.exports = require('express').Router()
 
 api
   .get('/heartbeat', (req, res) => res.send({ok: true}))
   .use('/auth', require('./auth'))
   .use('/users', require('./users'))
 
+api.get('/products', (req, res, next) => {
+    Thing.findAll({})
+    .then(products => res.send(products))
+    .catch(next)
+    //res.send("HELLOOOOOOO")
```

Comment:
Clean up old test code! Just get rid of line 14 so you're merging clean code to master!